package.json

```diff
@@ -92,29 +96,68 @@
     "watch:build": "tsc -p tsconfig.dist.json -w",
     "sourcegraph:prepublish": "npm run typecheck && npm run test && npm run build"
   },
+  "release": {
+    "branch": "latest"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
```

Comment:
`semantic-release` and `commitlint` don't make sense for an extension because an extension doesn't have semantic versions. You can remove the config and dependencies.

Reply:
Gotcha. Where did you see `semantic-release`?

Reply:
The `release` config above